go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/openconfig/magna
+
+go 1.16
```

Comment:
1.18

Reply:
I'm fine with making this change - but this is the minimum required version, if we always make this the latest then we mandate that folks move to that version even if we don't have a need for it. Is that intended as a result of your change, or just that "this is currently being written with 1.18, so there's no need to be compatible with 1.16"?